diff --git a/node_modules/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt b/node_modules/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt
index a1da771..1bcb461 100644
--- a/node_modules/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt
+++ b/node_modules/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt
@@ -71,6 +71,10 @@ object ExpoReactHostFactory {
         handler.onReactInstanceException(useDeveloperSupport, error)
       }
     }
+
+    override fun getReactNativeConfig(): com.facebook.react.fabric.ReactNativeConfig {
+      return com.facebook.react.fabric.ReactNativeConfig.DEFAULT_CONFIG
+    }
   }
 
   @OptIn(UnstableReactNativeAPI::class)
