diff --git a/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeAppModule.ts b/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeAppModule.ts
index f315f05..4e4b30c 100644
--- a/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeAppModule.ts
+++ b/node_modules/react-native-google-mobile-ads/src/specs/modules/NativeAppModule.ts
@@ -16,24 +16,23 @@
 
 import type { TurboModule } from 'react-native';
 import { TurboModuleRegistry } from 'react-native';
-import type { CodegenTypes } from 'react-native';
 
 export interface Spec extends TurboModule {
   // App initialization
   initializeApp(
-    options: CodegenTypes.UnsafeObject,
-    appConfig: CodegenTypes.UnsafeObject,
-  ): Promise<CodegenTypes.UnsafeObject>;
+    options: Object,
+    appConfig: Object,
+  ): Promise<Object>;
   setAutomaticDataCollectionEnabled(appName: string, enabled: boolean): void;
   deleteApp(appName: string): Promise<void>;
 
   // Events
   eventsNotifyReady(ready: boolean): void;
-  eventsGetListeners(): Promise<CodegenTypes.UnsafeObject>;
+  eventsGetListeners(): Promise<Object>;
   eventsPing(
     eventName: string,
-    eventBody: CodegenTypes.UnsafeObject,
-  ): Promise<CodegenTypes.UnsafeObject>;
+    eventBody: Object,
+  ): Promise<Object>;
   eventsAddListener(eventName: string): void;
   eventsRemoveListener(eventName: string, all: boolean): void;
 
@@ -42,15 +41,15 @@ export interface Spec extends TurboModule {
   removeListeners(count: number): void;
 
   // Meta
-  metaGetAll(): Promise<CodegenTypes.UnsafeObject>;
+  metaGetAll(): Promise<Object>;
 
   // JSON
-  jsonGetAll(): Promise<CodegenTypes.UnsafeObject>;
+  jsonGetAll(): Promise<Object>;
 
   // Preferences
   preferencesSetBool(key: string, value: boolean): Promise<void>;
   preferencesSetString(key: string, value: string): Promise<void>;
-  preferencesGetAll(): Promise<CodegenTypes.UnsafeObject>;
+  preferencesGetAll(): Promise<Object>;
   preferencesClearAll(): Promise<void>;
 }
 
